docs/man/man1/openshift-infra-f5-router.1

```diff
@@ -24,6 +24,14 @@ You may restrict the set of routes exposed to a single project (with \-\-namespa
 
 .SH OPTIONS
 .PP
+\fB\-\-allow\-wildcard\-routes\fP=false
+    Allow wildcard host names for routes
+
+.PP
+\fB\-\-allowed\-domains\fP=[]
```

Comment:
This is what the router admin would setup - allowed domains for this router instance. 
E.g.  `acme.auto, acme.test, acme.com`. 
The routes that users create would need to be in that domain set ala `www.dev.acme.test`.